Documentation/ColumnsConfig/Type/Select/CheckBox/Properties.rst

```diff
@@ -26,6 +25,11 @@ Select properties
 *  :ref:`items <columns-select-properties-items>`
 *  :ref:`MM <columns-select-properties-mm>`
 
+.. note::
```

Comment:
I would not add a note here. We are in select->checkbox (also for the other two). But a note would be suitable on the exclusiveKeys page:
https://docs.typo3.org/m/typo3/reference-tca/12.4/en-us/ColumnsConfig/Type/Select/Properties/ExclusiveKeys.html

Reply:
Done.